src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -19,15 +19,22 @@ namespace Microsoft.ML
     public static class StandardLearnersCatalog
     {
         /// <summary>
-        ///  Predict a target using logistic regression trained with the <see cref="SgdBinaryTrainer"/> trainer.
+        /// Predict a target using a linear classification model trained with <see cref="SgdBinaryTrainer"/>.
```

Comment:
>linear classification [](start = 37, length = 21)

Feels like word `calibrated` should be somewhere here.

Reply:
```
 /// The trained model can produce probability by feeding the output value of the linear
 /// function to a <see cref="PlattCalibrator"/>.
```
or this one

---
In reply to: [259444565](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259444565) [](ancestors = 259444565)

Reply:
I added this to SgdBinaryTrainer docs. We've been pushing all the detailed remarks to the Trainer/Estimator class so that the info is shared across all the different overload versions.

---
In reply to: [259445278](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259445278) [](ancestors = 259445278,259444565)